src/compile/common.ts

```diff
@@ -18,7 +18,7 @@ export function signalOrValueRef<T>(value: T | SignalRef): {value: T} | SignalRe
   if (isSignalRef(value)) {
     return value;
   }
-  return {value};
+  return value !== undefined ? {value} : undefined;
```

Comment:
If value is null, do we want {value: null}?

Reply:
Yes